### ISSUE TITLE
OS Optimization Doc Changes

### ISSUE THREAD

**Issue by ghost** [mastercoms/tf2cfg#730]
Please do not suggest users to use TCP optimizer it does mostly nothing for UDP traffic and can actually hurt networking performance seen here;
https://github.com/djdallmann/GamingPCSetup/blob/master/CONTENT/RESEARCH/NETWORK/README.md#networkthrottlingindex

https://stackoverflow.com/questions/60159716/how-to-enable-tcp-bbr-on-windows
A TCP change that actually helps users that are on Windows 11.

Article about Nagle's algorithm: http://ithare.com/tcp-peculiarities-as-applied-to-games-part-ii/

https://www.thewindowsclub.com/window-auto-tuning-in-windows-10
Depending on a Users connection enabling or disabling could help or hurt them.

Latency Mon should not be suggested to measure driver latency because it produces misleading results. Instead users should use xperf to measure DPC and ISR latency.

xperf can be grabbed here: https://learn.microsoft.com/en-us/windows-hardware/test/wpt/

Issues with Latency mon
No time out or delay before recording.
It doesn't pick up isrs or dpcs properly.
No detailed stats for individual drivers.
The KM driver generates a ton of dpcs alone creating invalid results.
The average measured latency is invalid because of this.

**Comment by Niterux** [comment]
The advice to use NVCleanstall should be removed, it almost completely bricked my NVIDIA drivers, started causing a ton of games to crash and my monitors started turning on and off...

**Comment by mastercoms** [comment]
Yup, I was already going to remove it, given what I've been seeing with it being too prone to user error. It has a lot of confusing options and lots of people won't get it right, and it doesn't have much benefit anyway, especially with the new NVIDIA App.
